aea/cli/scaffold.py

```diff
@@ -187,11 +186,13 @@ def scaffold_item(ctx: Context, item_type: str, item_name: str) -> None:
         with config_filepath.open("w") as fp:
             loader.dump(config, fp)
 
-        # update 'PUBLIC_ID' variable with the right public id
-        init_module = Path(dest, "__init__.py")
-        init_module.write_text(
-            re.sub(SCAFFOLD_PUBLIC_ID, str(new_public_id), init_module.read_text())
-        )
+        # update 'PUBLIC_ID' variable with the right public id in all python files!
+
+        for file_path in glob.glob(str(Path(dest, "*.py"))):
```

Comment:
We're not looking for any file. We're looking for init file only..

Reply:
now fixes only for `__init__.py` and `connection.py`